### DISCUSSION TITLE
"License not specified" vs "No license specified" vs "Unlicensed"

### DISCUSSION THREAD

**Discussion by lehors** [lfai/model_openness_tool#22] [General]
We currently have a bit of a mess on our hands with several variations of the same concept under different names. The existing models use "License not specified" (which we got from the original spreadsheet) and the forms provide the user with "No license specified" and for some fields "Unlicensed".

Given that we have more occurrences of "License not specified" than "No license specified" which merely appears in a few places in the code, I think we should settle on "License not specified".

Given that there is a license called "The Unlicense", I find the name "Unlicensed" confusing so I think we should avoid it. Besides, I'm not sure why this is added to some fields as an "extra license" (I see this is done in [mof.settings.yml](../blob/main/web/modules/mof/config/install/mof.settings.yml)). @gregcube @matthew-d-white what was the rationale for this? I would just remove it.

**Comment by lehors** [comment]
I submitted PR #23 to address this.
Running it locally, I can't see that anything is lost but if the "Unlicensed" extra license is really needed I will revert that part of my PR.

**Comment by gregcube** [comment]
I agree that this area could use a clean-up.  I'm fine with standardizing on `License not specified` for consistency, and removing `Unlicensed` to avoid confusion. I can't remember the exact reason or why Unlicensed was added; looking back I can't find those discussions.  I think it was a result of mixed values in the spreadsheet.

**Comment by lehors** [comment]
Ok so I think we're better off cleaning up the spreadsheet next time around we import one. Thanks.

**Comment by lehors** [comment]
This has been fixed
